src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -184,6 +189,30 @@ private String commandStringGenerator(
     return commandString.toString();
   }
 
+  /**
+   * Reads the docker server configuration from file ssh.rc under "/pathToResourceDocker".
+   */
+  private void readServerConfig() {
+    String configFilePath = pathToResourceDocker + "/ssh.rc";
+    try {
+      List<String> lines = Files.readAllLines(Paths.get(configFilePath), Charset.forName("UTF-8"));
+      for (String line : lines) {
+        if (line.contains("p2p_server=") && !line.endsWith("=")) {
+          setKeyServerIp(line.split("=")[1]);
+        } else if (line.contains("http_port=") && !line.endsWith("=")) {
+          setKeyServerHttpPort(line.split("=")[1]);
+        }
```

Comment:
I see it like this: 
We have to add a method that checks if the Server on that port is available, when not, a error must appear in the GUI where the Server-settings can be changed. Same if the Properties are empty. 
But this method here is only to read whats in the file, to reload the settings from the last session. The File is extracted when the application starts, therefor it exists.